laurent22/joplin
# Mobile: Fixes #10191: message for creating a note in the trash folder is not shown

## Summary

- Fixes #10191

Shows "There are no notes here." message in the trash folder and its sub folders when they are empty.

## Rationale

Got the id of the current selected folder by parsing noteSource state. Then got the corresponding folder from the folders state. If the items array length is 0 and if the current folder is

1.  the trash folder
2.  a deleted folder (sub folder of trash folder)

then show the message "There are no notes here."

## Demo


https://github.com/laurent22/joplin/assets/97161173/10829a39-94c7-4b3f-a606-825da61b01d0




## Testing
Tested on Android 13 physical device.
1. Make sure the trash folder is empty and then open it, it should display the "There are no notes here." message.
2. Create a notebooks and delete it without adding any note/todo.
3. Open that notebook in the trash folder,it should display "There are no notes here." message.


@PackElend label me please!
